src/Constraint/DraftFour/Type.php

```diff
@@ -13,6 +13,26 @@
     const KEYWORD = 'type';
 
     /**
+     * Whether examples like 98249283749234923498293171823948729348710298301928331
+     * and "98249283749234923498293171823948729348710298301928331" are valid strings.
+     */
+    const BIGINT_MODE_STRING_VALID = 1;
```

Comment:
Since there are only 2 states, what about using a boolean?

```php
$type->allowLargeNumericStrings(true);
```

Not sure on the name.